paper-input.js

```diff
@@ -134,15 +134,15 @@ Polymer({
       }
 
       input::-webkit-input-placeholder {
-        color: var(--paper-input-container-color, var(--secondary-text-color));
+        color: var(--paper-input-container-input-placeholder-color, var(--paper-input-container-color, var(--secondary-text-color)));
```

Comment:
please add to documentation at the top

Reply:
@e111077  @ChrisBrownie55 Can this be made in a CSS mixin? I really need to be able to modify more than color ..

Reply:
Oh.. I see. But CSS Shadow Parts are not yet implemented in any polymer component or browser right? Isn't it better to use @apply than nothing until we have another option available?